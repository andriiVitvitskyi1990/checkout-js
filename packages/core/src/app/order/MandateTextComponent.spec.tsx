import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import React from 'react';

import { MandateTextComponent, MandateTextComponentProps } from './MandateTextComponent';

describe('MandateTextComponent',  () => {
    const props: MandateTextComponentProps = {
        mandateText: {
            account_holder_name: 'Test',
            iban: '1343442323',
            payment_reference: 'GDU856858',
        },
        methodId: 'ratepay',
        providerId: 'paypalcommercealternativemethod'
    }
    it('renders component with provided props', () => {
        const { container } = render(<MandateTextComponent {...props}/>);

        expect(container).toMatchSnapshot();
    });
});
