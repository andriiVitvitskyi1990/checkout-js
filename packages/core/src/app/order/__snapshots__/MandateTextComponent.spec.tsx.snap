// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MandateTextComponent renders component with provided props 1`] = `
<div>
  <ul
    data-test="order-confirmation-mandate-mandateText-list"
  >
    <li>
      :
      <b>
         Test
      </b>
    </li>
    <li>
      :
      <b>
         1343442323
      </b>
    </li>
    <li>
      :
      <b>
         GDU856858
      </b>
    </li>
  </ul>
</div>
`;
